lib/content-services/src/lib/document-list/components/document-list.component.html

```diff
@@ -25,7 +25,7 @@
     [class.adf-datatable-gallery-thumbnails]="data.thumbnails">
 
     <ng-template let-col>
-        <adf-filter-menu class="adf-filter-menu" [col]="col"></adf-filter-menu>
+        <adf-search-header [col]="col"></adf-search-header>
```

Comment:
Could you please add here something more generic here? in this way we don't couple document list and the search header.